src/components/dev-hub/article-share-footer.js

```diff
@@ -52,7 +52,7 @@ const ArticleShareFooter = ({ tags, title, url, ...props }) => {
                         </Link>
                     }
                 >
-                    Article link copied to clipboard!
+                    {tooltipText || 'Article link copied to clipboard!'}
```

Comment:
For Podcasts, let's link to this page on the developer hub, just like we do for articles. I do like this change to make the text customizable!